PIL/_imaging.pyi

```diff
@@ -0,0 +1,91 @@
+from PIL.aliases import *
+from typing import Optional, List, Text, Any
+
+PILLOW_VERSION = ...
+DEFAULT_STRATEGY = ...
+FILTERED = ...
+HUFFMAN_ONLY = ...
+RLE = ...
+FIXED = ...
+
+# List of functions is incomplete
+
+def alpha_composite(core1: ImagingCore, core2: ImagingCore) -> ImagingCore: ...
```

Comment:
Another case for mypy note noticing this, at least here. Which version are you running?